packages/Panel/README.md

```diff
@@ -30,29 +30,29 @@ npm install @paprika/panel
 | getPushContentElement | func                                                                                        | false    | null                          | Function that provides the container DOM element to be pushed.                                                                                  |
 | groupOffsetY          | number                                                                                      | false    | 0                             | Y offset that is passed down from <Panel.Group>                                                                                                 |
 | height                | [string,number]                                                                             | false    | "33%"                         | The height of the open Panel (when slide in from bottom)                                                                                        |
-| isCompact             | bool                                                                                        | false    | false                         | Control the compactness of the Panel                                                                                                            |
+| size                  | [undefined]                                                                                 | false    | Panel.types.sizes.MEDIUM      | Control the size of the Panel                                                                                                                   |
```

Comment:
Not a deal breaker, but this is a minor, unfortunate regression.  It actually seems like the Panel README wasn't updated last time changes were made to the API, so the previous version was quite out of date, but  `[Panel.types.sizes.MEDIUM, Panel.types.sizes.LARGE]` was more helpful than `[undefined]`. 

There's too much good stuff in this PR to get hung up on this minor detail, and Paprika is being phased out anyway, but any idea why this may have happened?

Reply:
Found it: changed [`oneOfType` to `oneOf`](https://github.com/acl-services/paprika/pull/1305/files#diff-17a3f1d9f151dd443e3c4c25cf44910acced1b27f02b1a3b23380f8a0ea6ecf9R241)